include/deal.II/hp/dof_handler.h

```diff
@@ -691,10 +707,7 @@ namespace hp
   private:
 
     /**
-     * Copy constructor. I can see no reason why someone might want to use it,
-     * so I don't provide it. Since this class has pointer members, making it
-     * private prevents the compiler to provide it's own, incorrect one if
-     * anyone chose to copy such an object.
```

Comment:
I agree. The comment may be misleading, but there is a good reason not to implement the copy constructor.